content/v1.3/introduction/getting-started.md

```diff
@@ -491,12 +493,15 @@ We can verify it all works as expected with a unit test:
 
 RSpec.describe Book do
   it 'can be initialized with attributes' do
-    book = Book.new(title: 'Refactoring')
+    book = Book.new(title: 'Refactoring', author: 'Martin Fowler')
     expect(book.title).to eq('Refactoring')
+    expect(book.author).to eq('Martin Fowler')
   end
 end
 ```
 
+(Generally we recommend against 'testing the framework' like this, but it's useful here to demonstrate how Entities work in Hanami.)
```

Comment:
I'm not sure I feel this needs to be parenthesized.

Reply:
I changed this to a "convention" block, which seems appropriate. It's a (Green) aside now, which feels right. Users can test the framework if they want! I just wouldn't :)

Reply:
> It's a (Green) aside now, which feels right.

I'm not seeing this in the preview. Was this something you saw locally?

Reply:
Whoops, I had a typo :) https://github.com/hanami/guides/pull/93/commits/05d5f5c413ef228802eda843d18c8fb2fa61d7cd